### DISCUSSION TITLE
A shortcut or ability to Expand more or less Artboard Layers at once.

### DISCUSSION THREAD

**Discussion by nopalfi** [uxbox/uxbox#1211] [Ideas]
Hi, i've been using Penpot for a few days and it has been great experience.
One thing that annoys me is sometimes after completed edit individual artboards, i need to expand less every single artboards. Will be there a feature that expand less all the artboard layers at once? Or it has been implemented somewhere.

**Comment by elhombretecla** [comment]
Hi @nopalfi :)

You can use "shift + click" on one artboard and all your artboards will be collapsed. This is what you mean?

![expand-layers](https://user-images.githubusercontent.com/1045247/133219232-8ed99276-6362-4e0f-a369-edc67b369cb4.gif)

**Reply by nopalfi** [reply]
YES!! Exactly like what i want.
Thank you for the respond :smile:
I do realize you didn't mention it in your youtube at "Workspace interface" video. That's why i didn't knew this trick. :laughing:

**Reply by elhombretecla** [reply]
You're right. We are going to add this shortcut to the [Help Center](https://help.penpot.app/user-guide/introduction/shortcuts/) :)

Glad to help!

**Comment by gmaggio** [comment]
I suggest an additional improvement, that is, to collapse only as far as the layer being clicked instead of collapsing all the way to the topmost.
